apps/web/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Listbox,
+  ListboxButton,
+  ListboxOptions,
+  Transition,
+} from "@headlessui/react";
+import { ChevronUpDownIcon } from "@heroicons/react/24/outline";
+
+import { Option } from "./components/Option";
+
+export interface Option {
+  value: string | number;
+  label: string;
+}
+
+export interface DropdownProps {
+  options: Option[];
+  selected: Option;
```

Comment:
I'd revert this to the previous prop and accept the index of one of the elements pass in the `options` props.

I think it was simpler to set